components/scream/src/physics/p3/p3_functions_f90.cpp

```diff
@@ -3559,22 +3576,25 @@ void p3_main_f(
   // Sync back to host
   Kokkos::Array<view_2d, P3MainData::NUM_ARRAYS - 10> inout_views = {
     qc_d, nc_d, qr_d, nr_d, qitot_d, qirim_d, nitot_d, birim_d, qv_d, th_d,
-    diag_ze_d, diag_effc_d, diag_effi_d, diag_vmi_d, diag_di_d, diag_rhoi_d, mu_c_d, lamc_d, cmeiout_d, prain_d, nevapr_d, prer_evap_d, pratot_d, prctot_d, liq_ice_exchange_d, vap_liq_exchange_d, vap_ice_exchange_d, rflx_d, sflx_d, prt_liq_temp_d, prt_sol_temp_d
+    diag_effc_d, diag_effi_d, diag_rhoi_d, mu_c_d, mu_r_d, cmeiout_d, prain_d,
+    nevapr_d, prer_evap_d, liq_ice_exchange_d, vap_liq_exchange_d,
+    vap_ice_exchange_d, rflx_d, sflx_d, prt_liq_temp_d, prt_sol_temp_d
   };
   Kokkos::Array<size_t,  P3MainData::NUM_ARRAYS - 10> dim1_sizes_out;
   Kokkos::Array<size_t,  P3MainData::NUM_ARRAYS - 10> dim2_sizes_out;
   for (size_t i = 0; i < P3MainData::NUM_ARRAYS - 10; ++i) dim1_sizes_out[i] = ni;
   for (size_t i = 0; i < P3MainData::NUM_ARRAYS - 10; ++i) dim2_sizes_out[i] = nk;
 
 
-  dim2_sizes_out[27] = nk+1; // rflx
-  dim2_sizes_out[28] = nk+1; // sflx
-  dim1_sizes_out[29] = 1; dim2_sizes_out[29] = ni; // prt_liq
-  dim1_sizes_out[30] = 1; dim2_sizes_out[30] = ni; // prt_sol
+  dim2_sizes_out[21] = nk+1; // rflx
+  dim2_sizes_out[22] = nk+1; // sflx
+  dim1_sizes_out[23] = 1; dim2_sizes_out[23] = ni; // prt_liq
+  dim1_sizes_out[24] = 1; dim2_sizes_out[24] = ni; // prt_sol
 
   pack::device_to_host({
-      qc, nc, qr, nr, qitot, qirim, nitot, birim, qv, th,
-      diag_ze, diag_effc, diag_effi, diag_vmi, diag_di, diag_rhoi, mu_c, lamc, cmeiout, prain, nevapr, prer_evap, pratot, prctot, liq_ice_exchange, vap_liq_exchange, vap_ice_exchange, rflx, sflx, prt_liq, prt_sol
```

Comment:
same here with lamc -> mu_r (just marking everywhere I notice it just in case you need to change it and would appreciate knowing where a few might be :-)

Reply:
Thanks! Appreciated.